packages/react-components/src/useConnect.ts

```diff
@@ -35,21 +35,23 @@ export function useConnect(
 ): Connect {
     const [isConnecting, setIsConnecting] = useState(false);
     const [connectError, setConnectError] = useState('');
-    const connect = useCallback(() => {
+    const connect = useMemo(() => {
         if (!connector) {
-            throw new Error('no connector to connect from');
```

Comment:
It would be nice to leave this as a comment:

`No connector to connect from, so this hook returns 'undefined' for the 'connect' function in that case` or leave something in the comment description of the `useConnect` hook, when the `connect` function is `undefined`

Reply:
Done.